package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Intercept imports in Node.js",
   "main": "index.js",
   "scripts": {
-    "test": "c8 --check-coverage --lines 90 imhotap --runner test/runtest --files test/{hook,low-level,other}/*",
-    "test-win": "c8 --check-coverage --lines 90 imhotap --runner test\\runtest.bat --files test/{hook,low-level,other}/*",
+    "test": "c8 --check-coverage --lines 89 imhotap --runner test/runtest --files test/{hook,low-level,other}/*",
```

Comment:
I hate this PR...
I have reduced the coverage number, because I have added some NODE_VERSION related code, it means that it is code that can be played in some node version and the coverage can't increase in that jobs.

Reply:
It's possible to merge reports so that doesn't happen, but that's out of the scope of this PR.